docs/KubeFATE_command_line_user_guide.md

```diff
@@ -0,0 +1,279 @@
+# KubeFATE Command Line User Guide
+
+### What is KubeFATE command line
+
+KubeFATE command line is a command line tool that connects KubeFATE  service to deploy FATE on Kubernetes.
+
+### Install KubeFATE command line
+
+Before using kubefate command line, you need to deploy KubeFATE  service.
```

Comment:
kubefate -> `kubefate`. Make sure we have cleared different objects mentioned in document:
1. Project name, be consistent with the spelling: KubeFATE;
2. Command line, make sure all wrap as code block: `kubefate`